### ISSUE TITLE
Impossible to configure OpenLDAP

### ISSUE THREAD

**Issue by alozano-sct** [nilsteampassnet/TeamPass#4523]
Hello, I need help to configure OpenLDAP.
I have installed TeamPass from 0, and configured LDAP, when I try it it tells me that the user connects correctly. But it never synchronizes my users nor does it let me log in with the LDAP user.
When doing the test to check the configuration, in the log of my LDAP server I do see the call.
I have tried debugging the LDAP log output in TeamPass, but it does not write anything.
In the user part, when synchronizing LDAP, a Caution appears, but it is blank, it does not give an error code or anything. 
I'm quite frustrated with the topic. 
I add screenshots of the LDAP configuration.
![LDAP_TEAMPASS](https://github.com/user-attachments/assets/a85e7b53-b441-4069-bed4-f5b9304d8baf)
![TeamPass_LDAP](https://github.com/user-attachments/assets/c5a498fa-8298-4380-a9a6-f44b83c4fa58)
![LDAP_TEAMPASS_SYNC](https://github.com/user-attachments/assets/7490ab6b-01eb-4197-9b27-c21ae4426143)

**Comment by DobroFenix** [comment]
Hi!
Use full CN user name

**Comment by alozano-sct** [comment]
@DobroFenix Thanks for comment, but this solution didnt work for me. I tried with a new service user to connected to LDAP, but nothing change, i have the same error. I tested the connection with LDAPsearch, and from CLI i can see the LDAP users withou any problems.

**Comment by rudival591** [comment]
yes, It follows as shown in the image I posted, it only worked in this configuration

**Comment by alozano-sct** [comment]
> yes, It follows as shown in the image I posted, it only worked in this configuration

But, its worked with ActiveDirectory, with OpenLdap not worked
